### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by Tomahna**
Use ResourceBundle in phoenicis-wine
See #321

### CODE REVIEW

File: phoenicis-wine/src/main/java/com/playonlinux/framework/Wine.java [hunk 1]
```diff
@@ -69,12 +70,46 @@
 @ScriptClass
 public class Wine implements SetupWizardComponent {
     private static final Logger LOGGER = Logger.getLogger(Wine.class);
-    private static final Architecture DEFAULT_ARCHITECTURE = Architecture.I386;
+
+    private static final String EXCEPTION_PREFIX_INCOMPATIBLE_ARCH = "A 32bit wineprefix cannot execute 64bits executables!"; //$NON-NLS-1$
```
Comment (qparis) [inline phoenicis-wine/src/main/java/com/playonlinux/framework/Wine.java]: I'm not a fan of using constant for log strings. It makes the code really hard to read.

**Comment by Tomahna** [comment]
I agree that the big series of constant definitions make the class looks scary, but the problem is more about the class itself which is probably too big. 
Whether it is more readable to read constants or string is subjective and i'm pretty much unopinionated on that matter.
But if i revert theses changes i'll have a lot of //$NON-NLS-1$ inside the code which might not help readability.

**Comment by qparis** [comment]
Agreed but I still think that string constant should not be used for log messages

**Comment by Tomahna** [comment]
Removed constants for log messages and exception messages

**Comment by qparis** [comment]
ok to test

**Comment by Veske** [comment]
Merge this one also?

**Comment by qparis** [comment]
The PR needs to be adapted with the new injection system

**Pull request closed without merging**